### DISCUSSION TITLE
The school you selected does not appear to have a campus location in your country

### DISCUSSION THREAD

**Discussion by ligen131** [community/community#42174] [GitHub Education] [answered]
Hi,

I am an undergraduate student at Huazhong University of Science and Technology. In the GitHub Global Campus, Huazhong University of Science and Technology(HUST)(华中科技大学) is regarded as having no campus location in mainland China, but in fact it is one of the world-recognized universities, and its campus location can be found on the map. Its address is Luoyu Road 1037, Hongshan District, Wuhan City, Hubei Province, China. This makes me have to use another email address instead of the email address with our school name suffix @hust.edu.cn to initiate this application.

My application to GitHub Global Campus was rejected for this reason. You can tell from my submitted IP address and location that I am submitting this application from school and that I am not connected to any VPN tools to submit this application. I also have completed [my GitHub user profile](https://github.com/ligen131) with my full name and a short bio. I certify that the information I have submitted is true and credible. Please help me approve this application. Thanks.

PS: My application was rejected for the following reasons:

> - The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
> - You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.
> - You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
> - The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

**Comment by ColinLeverger** [comment]
Hey, same problem here - for a different school in a different location! How to fix the location?

**Comment by samus-aran** [comment]
Hi @ligen131 👋 

I can find Huazhong University of Science and Technology, Hubei, China as part of the GitHub Education student application when searching for a school campus. Are you able to explain your steps on the application process ?

**Reply by ligen131** [reply]
Hello! @samus-aran  Thanks for your reply.

I chose the email address with the suffix @hust.edu.cn on the [home page of GitHub Education](https://education.github.com/discount_requests/pack_application) to apply. After I submit the photo in the next step, the system will return me the following information:

```
The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
```

So I have to use another email address instead of the email address with our school name suffix @hust.edu.cn to initiate this application.

But still, if I choose the school provided by the system, I get the above information after submitting the photo in the next step.

**Comment by custard-pi** [comment]
Same problem, applied for GitHub education in the campus, was told my school doesn't have a campus location in my country.😂

**Comment by ligen131** [comment, marked as answer]
I applied it again at school. After I took a photo, uploaded my student ID card and submitted it, the system passed my application within one minute. I was also surprised by this. I'm guessing that the possible reason is that the proxy was not completely closed in the last application. So please make sure that all your connections to github.com, education.github.com, and github.githubassets.com are directly through the school network when you apply, otherwise there is a high probability that the application will not pass.
